src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/ProjectDirectory.cs

```diff
@@ -6,21 +6,21 @@
 using System.IO;
 using System.Linq;
 using System.Threading;
-using Microsoft.AspNetCore.Testing;
 
 namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
 {
     internal class ProjectDirectory : IDisposable
     {
-#if PRESERVE_WORKING_DIRECTORY
-        public bool PreserveWorkingDirectory { get; set; } = true;
-#else
-        public bool PreserveWorkingDirectory { get; set; }
```

Comment:
Other than unusual debugging use cases (where someone doesn't want to add the define), does removing this property break anything?